icons/jest.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Jest icon</title><path d="M12 9.7l-1.4 2.9c.5.4.8 1 .9 1.7h1.1c.1-.6.4-1.2.9-1.6-.1 0-1.5-3-1.5-3z" fill="none"/><path d="M12.9 13.4L15.6 8l2.6 5.4c.1 0 .2-.1.3-.1L21 5.9H10.1l2.5 7.4c.1 0 .2 0 .3.1zm-.9 4.7c1 0 1.8-.8 1.8-1.8 0-.4-.1-.7-.3-1-.1-.1-.1-.2-.2-.3-.1-.1-.2-.1-.2-.2H13h-.1-.1-.1-.1-.1-.1-.1-.4-.1-.1-.1c-.8.2-1.3.9-1.3 1.7-.1.8.7 1.6 1.7 1.6zm7.4-3.5c-.1 0-.1 0 0 0h-.2-.1-.2-.1-.1-.1-.1-.1c-.1 0-.1.1-.2.1-.1.1-.2.1-.2.2-.1.1-.1.2-.2.3-.1.3-.2.6-.2.9 0 1 .8 1.8 1.8 1.8s1.8-.8 1.8-1.8c0-.8-.6-1.6-1.4-1.7-.3.2-.4.2-.4.2zM4.7 18.1c1 0 1.8-.8 1.8-1.8s-.8-1.8-1.8-1.8-1.7.9-1.7 1.8.8 1.8 1.7 1.8z" fill="none"/><path d="M19.5 9.3c-1.4 0-2.7-1-3-2.3h-1.2c-.3 1.3-1.6 2.3-3 2.3h-.5c-.4.6-.9 1.2-1.4 1.7-1 .9-2.1 1.6-3.3 2.1l-.3.1-.3-.2c-1.5-.7-2.2-2.2-2-3.7-.2 0-.3-.1-.5-.1 0 0 0 .1-.1.2-.4.9-.9 1.9-1.1 2.9-.2 1.1-.3 3.1 1.3 4.1.7.4 1.4.6 2.2.6 1.5 0 3.1-.8 4.9-1.7 1.3-.7 2.7-1.4 4.2-1.7.5-.1 1.1-.2 1.6-.3 1-.1 1.9-.3 2.7-.7.8-.5 1.4-1.2 1.5-2.1.1-.5 0-1.1-.1-1.6-.5.3-1.1.4-1.6.4z" fill="none"/><path d="M22.3 11.8c0-1.4-1-2.7-2.3-3L23 0H8.1l3 8.8c-1.3.4-2.2 1.6-2.2 3 0 1 .5 2 1.3 2.5-.3.4-.7.9-1.1 1.2-.8.7-1.6 1.3-2.6 1.6-.8-.6-1.2-1.6-.9-2.4 1.3-.4 2.2-1.6 2.2-3 0-1.7-1.4-3.1-3.1-3.1-1.7 0-3.1 1.4-3.1 3.1 0 .8.3 1.6.9 2.2 0 .1-.1.2-.1.3-.5 1-1 2-1.2 3.3-.4 2.5.3 4.5 1.9 5.6.8.6 1.8.9 2.9.9 1.8 0 3.7-.9 5.5-1.8 1.3-.6 2.6-1.3 3.9-1.6l1.5-.3c1.1-.2 2.2-.3 3.2-.9 1.2-.7 2-1.8 2.2-3.1.2-1 0-2-.4-2.9.2-.4.4-1 .4-1.6zm-1.4 0c0 1-.8 1.8-1.8 1.8s-1.8-.8-1.8-1.8c0-.3.1-.6.2-.9.1-.1.1-.2.2-.3l.2-.2c.1 0 .1-.1.2-.1h1.2c1 0 1.6.7 1.6 1.5zM10 1.4h11l-2.5 7.4c-.1 0-.2.1-.3.1l-2.6-5.4L13 8.9c-.1 0-.2-.1-.3-.1L10 1.4zm5 9.8c-.1-.7-.5-1.2-.9-1.7l1.4-2.9 1.5 3c-.4.4-.7 1-.9 1.6H15zm-3.5-1.1c.1 0 .1 0 0 0H13c.1.1.2.1.2.2.1.1.1.2.2.3.2.3.3.6.3 1 0 1-.8 1.8-1.8 1.8s-1.8-.8-1.8-1.8c.1-.6.7-1.3 1.4-1.5zm-6.8 0c1 0 1.8.8 1.8 1.8s-.8 1.8-1.8 1.8-1.8-.8-1.8-1.8.8-1.8 1.8-1.8zm16.2 6c-.2.9-.7 1.7-1.5 2.1-.8.4-1.7.6-2.7.7-.5.1-1.1.2-1.6.3-1.4.4-2.8 1-4.2 1.7-1.7.9-3.3 1.7-4.9 1.7-.8 0-1.5-.2-2.2-.7-1.5-1.1-1.5-3-1.3-4.1.2-1 .6-2 1.1-2.9 0-.1 0-.1.1-.2.2.1.3.1.5.1-.3 1.5.5 3 1.9 3.7l.3.1.3-.1c1.2-.4 2.3-1.1 3.3-2.1.6-.5 1-1.1 1.4-1.7h.5c1.4 0 2.7-1 3-2.3h1.2c.3 1.3 1.6 2.3 3 2.3.6 0 1.1-.2 1.6-.4.2.7.3 1.3.2 1.8z"/></svg>
```

Comment:
Looking at the quality of the icon it seems you've optimized a bit too much (look at the open circles at the bottom)